src/api/z3_api.h

```diff
@@ -4871,7 +4871,7 @@ extern "C" {
     /**
        \brief Return \c Z3_L_TRUE if \c a is true, \c Z3_L_FALSE if it is false, and \c Z3_L_UNDEF otherwise.
 
-       def_API('Z3_get_bool_value', INT, (_in(CONTEXT), _in(AST)))
+       def_API('Z3_get_bool_value', LBOOL, (_in(CONTEXT), _in(AST)))
```

Comment:
before LBOOL was an INT. Should it ever be a UINT?

Reply:
All other enums are casted to uint but right; int makes more sense